content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -176,49 +182,115 @@ For an unattended installation please use the following command:
 cmd /c 'msiexec /i openvswitch.msi ADDLOCAL="OpenvSwitchCLI,OpenvSwitchDriver,OVNHost" /qn'
 ```
 
-The installer propagates new environment variables. Please open a new command shell or logoff/logon to ensure the environment variables are refreshed.
+The installer propagates new environment variables. Please open a new command shell or logoff/logon to ensure the environment variables are refreshed. Make sure that the environment variables are refreshed by running the following command, and verifying the version is returned:
+```
+ovs-vsctl --version
+```
 
 For overlay, OVS on Windows requires a transparent docker network to function properly. Please use the following to create a transparent docker network which will be used by OVS. From powershell:
 ```
 docker network create -d transparent --gateway $GATEWAY_IP --subnet $SUBNET `
     -o com.docker.network.windowsshim.interface="$INTERFACE_ALIAS" external
 ```
-Where $SUBNET is the minion subnet which will be used to spawn pods on (the one which will be used by kubernetes), $GATEWAY_IP is the first IP of the $SUBNET and $INTERFACE_ALIAS is the interface used for creating the overlay tunnels (must have connectivity with the rests of the OVN hosts).
+Where $INTERFACE_ALIAS is the interface used for creating the overlay tunnels (must have connectivity with the rests of the OVN hosts). The variables $SUBNET and $GATEWAY_IP, and how to fetch their values is explained below.
```

Comment:
s/Where $INTERFACE_ALIAS/$INTERFACE_ALIAS

Also I think you should add a new line before: "The variables..."